public/scripts/main.js

```diff
@@ -142,32 +142,32 @@ function setup() {
 				button.innerHTML = name;
 				document.getElementById("audio-outs").appendChild(button);
 
+				setAudioOutput(name, button);
+
 				// set default (initial) volume (only for first audio device)
 				if (numOfAudioOuts == 1 && canGetVolume) {
 					// volume = devices[i].volume;
-					volume = 0; // temporary, need a way to get the volume
+					// volume = 0; // temporary, need a way to get the volume
 
 					// hack to get the current volume
-					$.ajax({
-						type: "GET",
-						url: "http://localhost:8004/" + devices[i].address + "/volume/get",
-						async: false,
-						headers: {
-							'Access-Control-Allow-Origin': '*'
-						},
-						success: function(data) {
-							console.log("setting volume to", data);
-							volume = data;
-							slider.setAttribute("value", volume);
-						},
-						contentType: "application/json; charset=utf-8"
-					});
+					// $.ajax({
```

Comment:
Yes. I don't believe the endpoint exists currently, it may have been moved somewhere else. In previous versions, this hasn't been executing because canGetVolume was always false; It's not set until after calling setAudioOutput(). In the few attempts I made, it was saying the address was invalid, so I decided it would be best to leave it out until the correct endpoint can be implemented.